Tools/dea_tools/classification.py

```diff
@@ -637,11 +697,17 @@ def feature_function(query):
     max_retries: int, default 3
         Maximum number of times to retry collecting samples. This number is invoked
         if the 'fail_threshold' is not reached.
+    time_field: str
```

Comment:
This sounds really useful!

For our Coastal Ecosystems project, we're using the same time range for all our training data (`time="2021"`). Can I confirm that if we leave `time_field` and `time_delta` to `None`, we can still provide our constant time period in our `dc_query` dict?

Reply:
Yes. These are the lines that update the `time` element of an odc query:

https://github.com/GeoscienceAustralia/dea-notebooks/blob/591feb9f30214b8f62466d632ef699e30575b004/Tools/dea_tools/classification.py#L491

If time_delta isn't provided, then the query timestamp doesn't get updated and remains as the constant value that you provided in the query.